### PR TITLE
Avoid explicitly using /workspace. Use dot (.) instead.

### PR DESCRIPTION
**Pull request by xuhdev**
- Avoid explicitly using /workspace . Use dot (.) instead.
- Do not create an additional layer for upgraded pip.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,5 @@ FROM continuumio/miniconda3:4.5.4
 WORKDIR /workspace
 RUN mkdir assets
 
-COPY requirements.txt /workspace
-RUN pip install --upgrade pip
-RUN pip install -r requirements.txt
-
-COPY . /workspace
+COPY . .
```
Comment (bdwyer2) [inline Dockerfile:6]: Can you please explain the purpose of this?

File: Dockerfile [hunk 2]
```diff
@@ -3,8 +3,5 @@ FROM continuumio/miniconda3:4.5.4
...
+RUN pip install --upgrade pip && pip install -r requirements.txt
```
Comment (bdwyer2) [inline Dockerfile:7]: This is great! It should reduce the size of all of our models.

File: Dockerfile (hunk 1 above)
Comment (xuhdev) [reply Dockerfile:6]: Remove redundancy because we have already specified `WORKDIR`. We would also reduce one layer that incorporates one `COPY` alone.

**Review by bdwyer2 — APPROVED** [review]

**Comment by xuhdev** [comment]
Can we merge this now?

**Review by MLnick — APPROVED** [review]
Seems fine to me - have we tested a downstream model with the updated image?

We should cut a new version of `max-base` after this change.

**Comment by xuhdev** [comment]
Yes, I tested with the Scene classifier.

**Pull request merged**